src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/credit_slips.yml

```diff
@@ -34,6 +34,7 @@ _admin_credit_slips_pdf_by_date:
     _legacy_link: AdminSlip:submitAddorder_slip
     _legacy_controller: AdminSlip
 
+# linter suggest changing to admin_credit_slips_index however the that same route already exist for [GET] method
```

Comment:
It's because this page has a custom behavior. Linter does not apply as the page is not standard. We need a way to tell the linter about unconventional pages, so it does not ring false alarms.